### DISCUSSION TITLE
My License is locked

### DISCUSSION THREAD

**Discussion by errrPuma** [aclap-dev/video-downloadhelper#161] [License & Purchase requests]
### What's your issue?

I formatted my PC and installed a new windows, when i put my license on Chrome is says is locked now.

### The 10 first digits (only 10) of your license number or email address

ipumarol13@gmail.com,

### I confirm that I didn't not paste my full license number

- [X] I confirm

**Comment by GrampaWildWilly** [comment]
Go into your VDH settings & expand the license section:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/587f9085-64ba-4c2b-8c53-76538fd2ebd1)

When you click the Recheck license button, that is supposed to send you an E-mail that gives you further instructions you need to follow.

This should be in the Wiki, @jcfrog Jérôme.

**Reply by jcfrog** [reply]
It's here: https://github.com/aclap-dev/video-downloadhelper/wiki#go-premium

**Reply by Trelligan** [reply]
I keep trying this over and over, and get no email. Please help?

**Reply by GrampaWildWilly** [reply]
That sounds like you registered VDH using one E-mail address but you now have a different one.  Since I don't work for VDH (I'm just another user), I can't fix this problem for you.  But @jcfrog Jérôme does work for VDH so he can fix it.

Also, the Wiki link you posted above, Jérôme, doesn't address the issue of the locked license.  It just describes how to register a license the first time.  The locked license deserves its own Wiki entry.

**Reply by devilsandro** [reply]
> Vá para as configurações do VDH e expanda a seção de licença:
> 
> ![#01](https://private-user-images.githubusercontent.com/75588964/305761119-587f9085-64ba-4c2b-8c53-76538fd2ebd1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMyNTQ5MTAsIm5iZiI6MTcyMzI1NDYxMCwicGF0aCI6Ii83NTU4ODk2NC8zMDU3NjExMTktNTg3ZjkwODUtNjRiYS00YzJiLThjNTMtNzY1MzhmZDJlYmQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODEwVDAxNTAxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY2ZDM3MzZkOWZiMjE4ZTg3Y2Y1MjI1ZDY0NmI5YmFiNmQ1MTU4MGQ2OTNkMjNmY2UxNGI0NjQyY2Q4YzQzMTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.av5oHKS3i8zaAlqtJi_lN6sI7K64pOPXlhBZRtPb7Lc)
> 
> Quando você clica no botão Recheck license, isso deve enviar um e-mail que lhe dá mais instruções que você precisa seguir.
> 
> Isso deve estar na Wiki, ?jcfrog Jérôme.

Goodnight! My license uses Mozilla Firefox and it says it is blocked after the extension is updated.

I'm not receiving emails to regularize
The first 10 digits (10 only) of your license number or email address

email: alexandreards@hotmail.com
license : be919b14ef
I confirm that I have not pasted my full license number

  X I confirm

**Comment by nala90** [comment]
Olá, minha licença apresenta esse erro, como posso proceder? 
![WhatsApp Image 2024-03-11 at 4 27 31 PM](https://github.com/aclap-dev/video-downloadhelper/assets/162072396/0852bd4c-20eb-42a0-85c7-4dec44774f23)

**Comment by nala90** [comment]
Olá, preciso de ajuda com minha licença não consigo fazer o registro

**Comment by devilsandro** [comment]
Goodnight! My license uses Mozilla Firefox and it says it is blocked after the extension is updated.

I'm not receiving emails to regularize
The first 10 digits (10 only) of your license number or email address

email: [alexandreards@hotmail.com](mailto:alexandreards@hotmail.com)
license : be919b14ef
I confirm that I have not pasted my full license number

X I confirm

**Reply by paulrouget** [reply]
@jcfrog ^

**Reply by jcfrog** [reply]
@devilsandro I guess you found our email, the license is unlocked.

**Comment by 1Ericat** [comment]
Hello there :)

 My license is blocked says my vdh when I tried to verify my code.

It says " License locked (check your email)" but I didn't get any .
Can someone help me with this please ?

my email: erikabalint79@yahoo.com
my license : 2aee45e62d
I confirm that I have not pasted my full license number

**Reply by jcfrog** [reply]
I guess you received our email, the license has been unlocked.

**Reply by 1Ericat** [reply]
Yes I did! Thank you very much !!!

**Reply by dipicorg** [reply]
What to do when my License is also blocked? Nothing changed for Years and now... 

Registration code 	
	2c85f8bef....

Granted to 	
	Christian Zuercher  (chr.zuercher@dipic.org)

Please Help

**Reply by jcfrog** [reply]
Hi,
Sorry, you should have received an email to reset your counter. Maybe in your spam box.
Anyway, I have reset your installation counter, should be OK now.

**Reply by dipicorg** [reply]
Hi and Sorry, the Purchase was not registered with the given Mail, it was amicol@vtxmail.ch  it is workinmg now  Thnx !

**Comment by sreverbe-re** [comment]
Hello everyone,
i reinstall addons for Edge, but when i clicked on Check license key, i received error: License locked (check your email).
I checked my email (included spam folder) but i don't received any email for unlock error.

Email: novellini-cassis0s@icloud.com
Key: c8828a3a........

Please someone help me.
Thanks

**Reply by jcfrog** [reply]
Sorry you did not receive our email, I unlocked your license.

**Reply by sreverbe-re** [reply]
Oh fantastic !
many many thanks jcfrog for very fast support.

greetings
sreverbe-re
